packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte

```diff
@@ -214,24 +214,16 @@
 	}
 
 	async function applyWidgetExample(example: Example, opts: ExampleRunOpts = {}): Promise<void> {
-		if (isLoading) {
-			return;
+		if ("text" in example) {
+			messages = [{ role: "user", content: example.text }];
+		} else {
+			messages = [...example.messages];
 		}
-		isLoading = true;
-		try {
-			if ("text" in example) {
-				messages = [{ role: "user", content: example.text }];
-			} else {
-				messages = [...example.messages];
-			}
-			if (opts.isPreview) {
-				return;
-			}
-			const exampleOutput = example.output;
-			await getOutput({ ...opts.inferenceOpts, exampleOutput });
-		} finally {
-			isLoading = false;
+		if (opts.isPreview) {
+			return;
 		}
```

Comment:
no need to manage `isLoading` in `aaplyWidgetExample` since it is already managed in `getOutput` function https://github.com/huggingface/huggingface.js/blob/13b95853a57718794b1bd21ccb522474a9a656b3/packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte#L197-L198

Reply:
So the initial issue was caused by `ìsLoading` not being properly reset to `false`, right?

Would be worth investigating how / why this happened

Reply:
I'm having a hard time connecting your changes to the initial issue (inference API not being called for examples after the first)

Can you explain why / how those changes fix it? 🙏

Reply:
> I'm having a hard time connecting your changes to the initial issue (inference API not being called for examples after the first)

You are right. This is not related to the issue described in this PR. https://github.com/huggingface/huggingface.js/pull/575#discussion_r1537613029 solves the issue completely. 

Just added this change since I've noticed this redundant `isLoading ` management. Can put it in another PR as well